homeassistant/components/fibaro/sensor.py

```diff
@@ -65,6 +71,8 @@ async def async_setup_entry(
     entities: list[SensorEntity] = []
     for device in hass.data[DOMAIN][entry.entry_id][FIBARO_DEVICES]["sensor"]:
         entities.append(FibaroSensor(device))
```

Comment:
If you add energy sensor as FibaroSensor the SensorStateClass.TOTAL_INCREASING is missing. As I understand this is important so that the sensor can be used in the energy panel. So either you need to extend FibaroSensor with the correct state class or you need to instantiate a FibaroEnergySensor and make it configurable to either get the value from energy or value attribute.